test/integration/targets/vmware_tag/tasks/tag_crud_ops.yml

```diff
@@ -0,0 +1,94 @@
+# Test code for the vmware_tag CRUD Operations.
+# Copyright: (c) 2019, Pavan Bidkar <pbidkar@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Wait for Flask controller to come up online
```

Comment:
After VMware test refactor, we have new syntax for handling vcsim and vcenter tests. Please take a look into https://github.com/ansible/ansible/blob/a62f0a2fdafc283203ff62a2a2de3c83916cdee1/test/integration/targets/vmware_host/tasks/main.yml#L5

Reply:
Thanks @Akasurde 
Done the changes